src/epcpm/parameterstointerface.py

```diff
@@ -1001,11 +1001,8 @@ def can_getter_setter_variable(can_signal, parameter, var_or_func_or_table):
             f"{can_signal.tree_parent.name}"
             f".{can_signal.name}"
         )
-    elif can_signal.tree_parent.tree_parent.name == 'CAN':
-        can_variable = (
-            f"&{can_signal.tree_parent.name}"
-            f".{can_signal.name}"
-        )
+    elif can_signal.tree_parent.tree_parent.name == "CAN":
+        can_variable = f"&{can_signal.tree_parent.name}" f".{can_signal.name}"
```

Comment:
If it's going to change it to one line, then might as well combine the two strings.
```suggestion
        can_variable = f"&{can_signal.tree_parent.name}.{can_signal.name}"
```

Reply:
[6b2e1ed](https://github.com/epcpower/m-pm/pull/3/commits/6b2e1ed67a8a09cb96ce172d2efe84cacaa6f2bc)